src/event/model/event_type.rs

```diff
@@ -7,7 +7,7 @@ pub enum EventType {
     #[serde(rename(
         serialize = "SetThreadStateNotificationEvent",
         deserialize = "discussion/comment/archive",
-        deserialize = "discussion/comment/restore"
+        deserialize = "discussion/restore"
```

Comment:
@inyono Review the actual semantics of the types. Might be that we don't trigger an event for all possible combinations of threads.